lib/connector/OAuthConnector.js

```diff
@@ -5,7 +5,34 @@ const authorizeView = require('fs').readFileSync(__dirname + '/authorize.html',
 });
 
 class OAuthConnector {
-    constructor() {}
+    constructor() {
+        /**
+         * Access tokens returned from ensureAccessToken method will expire not earlier
+         * than accessTokenExpirationBuffer milliseconds in the future.
+         */
+        this.accessTokenExpirationBuffer = 30000;
```

Comment:
Doesn’t JavaScript have a concept of class-level constants? Unless there’s parameterization to support changing these at runtime, can they be pulled out to be class-level static’s or coast’s.

Reply:
It does, but it is still an exprimental feature in the V8 bits in node 10, so it generates lots of warnings.